docs/getting_started.md

```
@@ -0,0 +1,354 @@
+
+If you are planning to use non-default values, make sure you set these values in the manifest.
+
+## Provision a Workload Cluster
+
+Once all providers and their dependencies are installed, you can provision a new cluster.
+
+### Prerequisites
+
+* DNS entries for the expected endpoints:
+  - `api.<baseDomain>`: The API endpoint for the cluster.
+  - `api.<baseDomain>`: The internal API endpoint for the cluster.
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)